pkg/rke/k8s_rke_system_images.go

```diff
@@ -9721,6 +9763,48 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.22.28",
 		},
 		// Enabled out of band post v2.7.10
```

Comment:
Please rebase as this got updated in another PR to state `Enabled out of band post v2.7.11` and ensure any newly added versions have the same comment.

Reply:
Rebased and verified.

Reply:
rebased and verified